docs/src/modules/components/AppFrame.js

```diff
@@ -127,9 +127,12 @@ const StyledAppBar = styled(AppBar, {
         width: 'calc(100% - 240px)',
       },
     }),
-    boxShadow: `inset 0px -1px 1px ${
-      theme.palette.mode === 'dark' ? theme.palette.primaryDark[700] : theme.palette.grey[100]
-    }`,
+    boxShadow: 'none',
+    borderStyle: 'solid',
+    borderColor:
+      theme.palette.mode === 'dark' ? theme.palette.primaryDark[700] : theme.palette.grey[100],
+    borderWidth: 0,
+    borderBottomWidth: 'thin',
```

Comment:
@michaldudak should it be `1px`?

Reply:
we have in `Divider`.

Reply:
Yeah, I wanted to make it consistent with the divider